### ISSUE TITLE
Show a "report bug" button on packages in openSUSE:Factory and other distro projects

### ISSUE THREAD

**Issue by Vogtinator** [openSUSE/open-build-service#14987]
**Is your feature request related to a problem? Please describe.**
Especially recently, users use comments on OBS packages as a place to report bugs. This is not the right way and will hopefully be disabled soon (https://github.com/openSUSE/open-build-service/issues/4192). However, this is a sign that somehow users gravitate towards those pages on OBS instead of bugzilla and we need a way to get them onto the right track again.

**Describe the solution you'd like**
Like for other packages on OBS where there is a bugowner defined, distro packages should also have a "report bug" button that ends up on bugzilla, with the appropriate assignee/cc (bugowner/maintainer from devel pkg/prj) and subject/text (project and package name) filled out as much as possible.

In the case that comments are disabled, the report bug button could be placed next to where the comments would be.

**Describe alternatives you've considered**
None.

**Additional context**
Most of https://github.com/openSUSE/open-build-service/issues/4192

**Comment by bmwiedemann** [comment]
An example link template could be
https://bugzilla.opensuse.org/enter_bug.cgi?classification=28&product=openSUSE+Tumbleweed&op_sys=openSUSE+Factory&cf_foundby=Community+User&short_desc=PACKAGENAME%3A+SUMMARY&comment=LONG+DESCRIPTION

**Comment by Vogtinator** [comment]
#14988 looks good, but there's one missing part: AFAICT the link would then use the same product/component as other packages, namely "openSUSE.org"  "3rd party software". There would have to be a way to configure a bugzilla link per project.

**Comment by Vogtinator** [comment]
> I think those other fields may be a bit out of scope for this bug, but it's not a bad idea to have a way to define additional parameters in projects

Yes, which is why I reported this as a feature. Until comments are disabled, ~noone will use the report bug link anyway...
